templates/zerver/api/fixtures.json

```diff
@@ -313,9 +313,10 @@
         "var_name": "content"
     },
     "nonexistent-stream-error": {
-        "code": "BAD_REQUEST",
+        "code": "STREAM_DOES_NOT_EXIST",
         "msg": "Stream 'nonexistent_stream' does not exist",
-        "result": "error"
+        "result": "error",
+        "stream": "nonexistent_stream"
```

Comment:
By the way, did our API tests catch the need to do this?  If so, awesome!!!

Reply:
Oh yeah, I just sent you a message about this! I am very happy with how this framework handles this now! :)